modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -50,16 +49,24 @@ spec:
       version: v1
       resource: deployments
 ----
-
-A service binding request is created and the {servicebinding-title} controller projects the database service connection information into the application deployment as files by using a volume mount. After the request is successful, the application is redeployed and the connection is established.
-
++
+A service binding request is created and if the database service connection request is successful, the application is redeployed and the connection is established.
++
 .Binding connector
 image::odc-binding-connector.png[]
-
++
 [NOTE]
```

Comment:
I realize this was not added by you, Dhruv, but this looks more like a `[TIP]` rather than a `[NOTE]`.

Reply:
Fixed.